src/main.go

```diff
@@ -95,12 +99,51 @@ func init() {
 	logger.Terminal("Bench-routes is up and running", "p")
 }
 
+func cleanup() {
+	logger.Terminal("No. of go routines running are: "+strconv.Itoa(runtime.NumGoroutine()), "p")
```

Comment:
in place of `logger.Terminal("No. of go routines running are: "+strconv.Itoa(runtime.NumGoroutine()), "p")`,
```
logger.Terminal(fmt.Sprintf("Alive %d go-routines", strconv.Itoa(runtime.NumGoroutine())), "p")
```